docs/index.asciidoc

```diff
@@ -50,6 +50,7 @@ This plugin supports the following configuration options plus the <<plugins-{typ
 | <<plugins-{type}s-{plugin}-gzip>> |<<boolean,boolean>>|No
 | <<plugins-{type}s-{plugin}-path>> |<<string,string>>|Yes
 | <<plugins-{type}s-{plugin}-write_behavior>> |<<string,string>>|No
+| <<plugins-{type}s-{plugin}-stale_cleanup_interval>> |<<number,number>>|No
```

Comment:
Thanks for adding the new option to the options list.  A lot of people forget to do this step. Please move this option so that it will be in alphabetical order.